linuxmint/20/cinnamon/install.sh

```diff
@@ -0,0 +1,108 @@
+#!/bin/bash
+
+#
+# This script is for Linux Mint 20 Ulyana Cinnamon Edition to install and
+# configure xrdp to be used with Hyper-V's Enhanced Mode.
+#
+#
+
+###############################################################################
+# Use HWE kernel packages
+#
+HWE=""
+
+###############################################################################
+# Update our machine to the latest code if we need to.
+#
+
+if [ "$(id -u)" -ne 0 ]; then
+    echo 'This script must be run with root privileges' >&2
+    exit 1
+fi
+
+apt update && apt upgrade -y
+
+if [ -f /var/run/reboot-required ]; then
+    echo "A reboot is required in order to proceed with the install." >&2
+    echo "Please reboot and re-run this script to finish the install." >&2
+    exit 1
+fi
+
+###############################################################################
+# XRDP
+#
+
+# Install hv_kvp utils
+apt install -y linux-tools-virtual${HWE}
+apt install -y linux-cloud-tools-virtual${HWE}
+
+# Install the xrdp service so we have the auto start behavior
+apt install -y xrdp
+
+systemctl stop xrdp
+systemctl stop xrdp-sesman
+
+# Configure the installed XRDP ini files.
+# use vsock transport.
+sed -i_orig -e 's/port=3389/port=vsock://-1:3389/g' /etc/xrdp/xrdp.ini
```

Comment:
```suggestion
sed -i_orig -e 's/port=3389/port=vsock:\/\/-1:3389/g' /etc/xrdp/xrdp.ini
```
Found a syntax error